### DISCUSSION TITLE
PSA: Project rename

### DISCUSSION THREAD

**Discussion by Eugeny** [Eugeny/tabby#4087] [General]
📢  The next release will come with a new name - **Terminus** becomes **Tabby**.

There are multiple reasons for this:

* **Terminus** is too close to **Termius** - a commercial terminal app. This causes a number of users to end up on our issue tracker when they're actually looking for help with **Termius** - it's better to cover your butt before the lawsuit comes than regret it later.
* There's another terminal app that's literally called "[Terminus](https://gitlab.com/rastersoft/terminus)" and is packaged under this name at least in Debian and Ubuntu repos.
* It just rolls off the tongue better 🐱 

---

**Update**: the new release will be up in a few hours. 

Good to know:

- After upgrade, your config file will be automatically copied to the new data directory (see https://github.com/Eugeny/tabby/wiki/Config-file)
- Plugins will keep working as-is
- I'll publish new versions of my own plugins under new names 

Plugin developers:

- New API packages will be released under new names (`tabby-*`) - you'll only need to update your `package.json` if you'll need to use new APIs.
- To make sure that your plugin stays discoverable in the future, replace the `terminus-plugin` keyword with `tabby-plugin` in `package.json`
- A newer `tabby-*` package will be treated as an upgrade to the corresponding `terminus-*` package.
- Installed plugins named `tabby-*` will override the corresponding `terminus-*` plugins.

**Comment by EricBright** [comment]
The website links/name/contents at https://eugeny.github.io/tabby/ need updating now too.

**Comment by MostHated** [comment]
This is good news. It was rather frustrating having to freeze the package so I could update my system without overwriting it if I installed it via deb in order to get the typical/better overall system experience of having a .desktop entry, icons, etc.

**Comment by fuomag9** [comment]
Finally I don't have to rename the deb package anymore! Also the name similarity between termius and terminus is how I discovered this terminal in the first place

**Comment by liaoke0123** [comment]
ok, fine, I just recommended "Terminus" to my friend
but you know ,  the first item is "Termius" in google and he didn't know it ...

**Comment by Zeragamba** [comment]
minor QoL thing; Can the windows .exe be named "Tabby Terminal" so that windows search can still pick it up as an option while searching for "term"

**Comment by syedak512** [comment]
hi
